src/test/resources/com/puppycrawl/tools/checkstyle/checks/naming/localfinalvariablename/InputLocalFinalVariableName.java

```diff
@@ -188,6 +188,26 @@ void veryLong()
           enough talk */
     }
 
+    // check for try-with-resources
+    void method() throws Exception {
+        final String fileName = "Test";
+        try (BufferedReader bR = new BufferedReader(new InputStreamReader(
+                new FileInputStream(fileName),"UTF-8"))) {
+        } finally {
+
+        }
+    }
+
+    void method2() throws Exception {
+       new java.util.Comparator() {
```

Comment:
this has nothing to do with try-with-resources so it can be removed.
If you need examples, see the first post in the main issue.